SQL/0000-00-00-schema.sql

```diff
@@ -237,6 +240,16 @@ CREATE TABLE `test_names` (
   CONSTRAINT `FK_test_names_1` FOREIGN KEY (`Sub_group`) REFERENCES `test_subgroups` (`ID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `test_names_fr` (
```

Comment:
this could be `test_name_dictionary` if `Full_name_fr` was replace by `alternative_name` and a `language_code` column was added referencing a `language` table row.
It could support any languages.